datacetner_clients.ymal

```diff
@@ -0,0 +1,50 @@
+version: '3.7'
```

Comment:
The filename and ending has "error". Can you change the name to "datacenter_clients.yaml" ?